controllers/session.go

```diff
@@ -66,3 +47,93 @@ func (c *ApiController) GetSessions() {
 		c.ResponseOk(sessions, paginator.Nums())
 	}
 }
+
+// GetSingleSession
+// @Title GetSingleSession
+// @Tag Session API
+// @Description Get session for one user in one application.
+// @Param   ID     query    string  true        "The ID(organization/application/user) of session"
+// @Success 200 {array} string The Response object
+// @router /get-session [get]
+func (c *ApiController) GetSingleSession() {
```

Comment:
You don't need to rename. The original function in controllers can be renamed to `GetBeegoSession()`, yours can be `GetSession()`